### PR TITLE
Ginkgo: Add support to kubernetes Beta and Alpha versions

### PR DESCRIPTION
**Pull request by eloycoto**
- Added support in `test/Vagrantfile` to alpha and Beta versions.
- Added another step on test-kubernetes jenkinsfile
- Fix some tables issues in docs

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

**Review by joestringer — APPROVED** [review]
Docs changes look good.

**Review by aanm — APPROVED** [review]

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
test-missing-k8s

**Comment by ianvernon** [comment]
test-missed-k8s

File: ginkgo-kubernetes-all.Jenkinsfile [hunk 1]
```diff
@@ -56,12 +56,41 @@ pipeline {
...
+            }
+        }
     }
     post {
         always {
             cleanWs()
             sh "cd ${TESTDIR}; K8S_VERSION=1.7 vagrant destroy -f || true"
             sh "cd ${TESTDIR}; K8S_VERSION=1.9 vagrant destroy -f || true"
+            sh "cd ${TESTDIR}; K8S_VERSION=1.10 vagrant destroy -f || true"
+            sh "cd ${TESTDIR}; K8S_VERSION=1.11 vagrant destroy -f || true"
```
Comment (ianvernon) [inline ginkgo-kubernetes-all.Jenkinsfile:93]: Just realized - will having 4 sets of K8s VMs running on the host at once be feasible? If the stages are run in serial order (i.e., 1.7 and 1.9 then 1.10 and 1.11), I would delete 1.7 and 1.9 VMs in that stage's `post` section vs. waiting until the end of the entire job.

**Review by ianvernon — CHANGES_REQUESTED** [review]

File: ginkgo-kubernetes-all.Jenkinsfile (hunk 1 above)
Comment (eloycoto) [reply ginkgo-kubernetes-all.Jenkinsfile:93]: Hi @ianvernon 

Servers are deleted in Ginkgo:
https://github.com/cilium/cilium/blob/master/test/test_suite_test.go#L194-L200

That lines are to make sure that no leftover VMS are in place, but Ginkgo uses to delete all ;-)

**Comment by eloycoto** [comment]
@ianvernon I think that this PR can be merged. The issues with Kubernetes Beta + Alpha need to be addressed in future issues.

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

File: ginkgo-kubernetes-all.Jenkinsfile (hunk 1 above)
Comment (ianvernon) [reply ginkgo-kubernetes-all.Jenkinsfile:93]: Ah, yes - makes sense :)

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
cilium/contributing was added as a review by GitHub, but there are no contributing docs in this PR (likely due to the commit being removed in this PR, or merged by a different PR). Thus, all necessary approvals are in for this to be merged. Merging this :-)

**Pull request merged**